### DISCUSSION TITLE
Where to Start for a New Language Driver

### DISCUSSION THREAD

**Discussion by Tembocs** [geldata/gel#3481] [Q&A]
I really see the potential of EdgeDB even at these early stages. If some people want to write drivers for languages that have no EdgeDB drivers yet what kind of steps would you advice they follow? Basically what are the references and steps they should look into to make the work more easier, less painful and interesting, just to encourage more language drivers written for EdgeDB?

**Comment by elprans** [comment]
We plan to write a "driver implementer's guide" soon, but for now the best way is to study the existing drivers and read the [protocol](https://www.edgedb.com/docs/reference/protocol/index) documentation.

Hop on [Discord](https://www.edgedb.com/p/discord), we can help you get started.

**Reply by Tembocs** [reply]
Noted, thank you.
